README.md

```diff
@@ -1 +1,177 @@
-# undercoat
\ No newline at end of file
+Undercoat.css is a **lightweight** framework with **no default theme** out of the box. So you **don't need to fight** against it!
+
+![Travis (.org)](https://img.shields.io/travis/undercoat/undercoat.svg?style=flat-square) ![GitHub release](https://img.shields.io/github/release/undercoat/undercoat.svg?style=flat-square) ![npm](https://img.shields.io/npm/v/undercoat.svg?style=flat-square)  ![npm](https://img.shields.io/npm/dt/undercoat.svg?style=flat-square) ![Twitter Follow](https://img.shields.io/twitter/follow/undercoatcss.svg?label=Follow&style=flat-square) ![WCAG 2.0 Level AA](https://img.shields.io/badge/WCAG_2.0-Level_AA-brightgreen.svg?style=flat-square) ![schema.org](https://img.shields.io/badge/microdata-schema.org-brightgreen.svg?style=flat-square)
+
+---
+
+**CAUTION! IT'S UNDER HEAVY DEVELOPMENT CURRENTLY! DON'T USE IT IN PRODUCTION!**
+
+---
+
+## Contents
+
+1. [Description](#Description)
+2. [Installation](#Installation)
+3. [Usage](#Usage)
+4. [Browser support](#Browser-support)
+5. [Key concepts](#Key-concepts)
+6. [Contributing](#Contributing)
+
+## Description
+
+Undercoat.css or just Undercoat is ready to use out of the box because it's written on vanilla CSS and doesn't require building. The main advantages are:
+
+- flexible and predictable architecture based on [BEM methodology](https://en.bem.info/methodology);
+- no need in overriding the framework styles because there's no default theme provided;
+- SEO-friendly semantic HTML markup enriched by [microdata](https://schema.org);
+- mobile-friendly by virtue of following of [Google recommendations](https://developers.google.com/search/mobile-sites) and using web standards capabilities;
```

Comment:
I think it would be better to leave just like this: 

following [Google recommendations]

Reply:
Thx!